server/views/pages/search/caSearch/caSearch.test.ts

```diff
@@ -262,6 +320,13 @@ describe('View CA Search Results', () => {
     expect($('#release-date-2').text()).toBe('CRD: 1 Aug 2025')
     expect($('#licence-last-worked-on-by-2').text()).toBe('Test Updater')
     expect($('#licence-status-2 > .status-badge').text().trim()).toBe('Active')
+
+    expect($('#name-3 > .search-offender-name > a').text()).toBe('Test Person 2')
+    expect($('#nomis-id-3').text()).toBe('A1234AB')
+    expect($('#probation-practitioner-3').text()).toBe('Unallocated')
```

Comment:
check: do we need a test checking `Not allocated yet` as well in the case of TS? Maybe I just can't see it here 😅